notebooks/ms2deepscore/tutorial.ipynb

```diff
@@ -97,7 +108,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 4,
```

Comment:
![image](https://user-images.githubusercontent.com/63718157/127297147-4436cafe-58ec-4996-b8a7-5bcbac8d90b1.png)
the token is still there and the dev endpoint should not be committed either

Reply:
and it is missing the ion mode as well
![image](https://user-images.githubusercontent.com/63718157/127297313-3d8a46df-4d79-4a06-8cd6-7cc0c31557b1.png)